src/main/bash/sdkman-install.sh

```diff
@@ -204,6 +204,12 @@ function __sdkman_checksum_zip() {
 		__sdkman_echo_yellow "Metadata file not found at '${headers_file}', skipping checksum..."
 		return
 	fi
+	
+	if [[ "$sdkman_checksum_skip" == 'true' ]]; then
+		echo ""
+		__sdkman_echo_green "Checksums are disabled, skipping verification..."
```

Comment:
This is pure informational output, so it should be printed in yellow. We use green for a successful outcome and red for a failed outcome.